packages/main/src/plugin/configuration-impl.ts

```diff
@@ -68,21 +72,24 @@ export class ConfigurationImpl implements containerDesktopAPI.Configuration {
 
   async update(section: string, value: unknown): Promise<void> {
     const localKey = this.getLocalKey(section);
-
     // now look if we have this value
     const localView = this.getLocalView();
 
+    const eventArg: ConfigurationChangedArg = {
```

Comment:
I'm not sure to follow the pattern `xxxArg`

AFAIK we're sending an event there, so `ConfigurationChangeEvent` might be more appropriated ?

Reply:
I would expect the value be part of the event as well ?